helix-core/src/test/java/org/apache/helix/controller/changedetector/TestResourceChangeDetector.java

```diff
@@ -431,8 +431,12 @@ public void testResetSnapshots() {
             .setResources(new HashSet<>(_dataAccessor.getChildNames(_keyBuilder.idealStates())))
             .setWaitTillVerify(TestHelper.DEFAULT_REBALANCE_PROCESSING_WAIT_TIME)
             .build();
-    Assert.assertTrue(_clusterVerifier.verify());
-
+    try {
+      boolean result = _clusterVerifier.verify();
```

Comment:
nit: Can we do 
Assert.assertTrue(_clusterVerifier.verify()); ?

Reply:
Changed.